{"created_at": "2024-02-03 18:12:13.784368", "build": null, "status": "unhealthy", "report": {"datastore_classic": {"desc": "Datastore Classic", "results": {"TestDatastoreClassic.test_healthy": {"desc": "Test API self-reports as healthy", "args": [], "outcome": "failed", "error_msg": "self = <test_datastore_classic.TestDatastoreClassic object at 0x7fcb9b49f210>\n\n    def test_healthy(self):\n        \"\"\"\n        Test API self-reports as healthy\n        \"\"\"\n        url = \"https://datastore.codeforiati.org/api/1/about/\"\n        resp = requests.get(url)\n        assert resp.status_code == 200\n        j = resp.json()\n>       assert j[\"status\"] == 'healthy'\nE       AssertionError: assert 'unhealthy' == 'healthy'\nE         - healthy\nE         + unhealthy\nE         ? ++\n\ntests/test_datastore_classic.py:19: AssertionError"}}, "status": "unhealthy"}, "iati_data_dump": {"desc": "IATI Data Dump", "results": {"TestIATIDataDump.test_metadata": {"desc": "Test metadata was created in the last 24 hours", "args": [], "outcome": "failed", "error_msg": "self = <test_iati_data_dump.TestIATIDataDump object at 0x7fcb9b42c2d0>\n\n    def test_metadata(self):\n        \"\"\"\n        Test metadata was created in the last 24 hours\n        \"\"\"\n        url = \"https://gist.githubusercontent.com/\" + \\\n              \"codeforIATIbot/efd190029713c6775c43962444dcb8df/\" + \\\n              \"raw/metadata.json\"\n        resp = requests.get(url)\n        assert resp.status_code == 200\n        j = resp.json()\n        created_at = j[\"created_at\"]\n        delta_since = (datetime.now(pytz.utc) - date_parse(created_at))\n>       assert delta_since.days == 0\nE       assert 13 == 0\nE        +  where 13 = datetime.timedelta(days=13, seconds=42863, microseconds=781437).days\n\ntests/test_iati_data_dump.py:23: AssertionError"}}, "status": "unhealthy"}}}